tests/data/input/dflowfm_individual_files/obsPoints_obs.ini

```diff
@@ -0,0 +1,30 @@
+# 2022-05-19: Taken from D-Flow FM UM, F.2.2.1, example 1.
+[General]
+fileVersion = 2.00
+fileType    = obsPoints
+
+[ObservationPoint]
+name         = ABDN
+x            = -2.06250000e+00
+y            = 5.71583333e+01
+
+[ObservationPoint]
+name         = AVMH
+x            = -2.73750000e+00
+y            = 5.15083334e+01
+
+[ObservationPoint]
+name         = #St Helier Jersey#
```

Comment:
Isn't something between two # a comment?

Reply:
No, this is our ugly Deltares-customized ini format: Single # is a comment char, whereas a strings enclosed in two #'s are string literal with possible whitespace. It is how it is... ;-)